plugins/poll/assets/javascripts/discourse/components/poll-buttons-dropdown.gjs

```diff
@@ -35,6 +35,20 @@ const buttonOptionsMap = {
     icon: "lock",
     action: "toggleStatus",
   },
+  showAbsolute: {
+    className: "btn-default toggle-absolute",
+    label: "poll.show-absolute.label",
+    title: "poll.show-absolute.title",
+    icon: "info",
+    action: "toggleAbsolute",
+  },
+  showPercentage: {
+    className: "btn-default toggle-absolute",
```

Comment:
Its unexpected the class name is "toggle-absolute" here

Reply:
done!